modules/ROOT/pages/twilio/twilio-connector.adoc

```diff
@@ -148,9 +148,9 @@ For more information on each format, see the See Also section at the end of this
 
 == Use Case: Studio
 
-In the following example, a Mule application sends a message to a phone number, and then redacts it.
+In the following example, a Mule application sends a message to a phone number, and then redacts it. This example uses variables for field values. You can provide the values for each variable in a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
```

Comment:
```suggestion
In the following example, a Mule application sends a message to a phone number, and then redacts it. 

Note that although this example uses variables for field values, you can provide actual values for each variable by using a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
```
Could the sentence "To use a properties file..." be omitted, given the cross-ref.?

Reply:
Yes to eliminating "To use a properties file." No to "although." It changes the technical meaning and is incorrect.